### PR TITLE
Add engine template for linkis web

### PR DESCRIPTION
**Pull request by Beacontownfc**
Shown below
4fe1a1c3fbc4ae7bd23fa6963bc5240

### CODE REVIEW

File: linkis-web/src/apps/linkis/module/engineConfigurationTemplate/index.vue [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+            </template>
+          </Table>
+        </Card>
+      </Col>
+    </Row>
+    <Modal
+      width="800"
+      class="modal"
+      v-model="modalShow"
+      title="修改"
```
Comment (jacktao007) [inline linkis-web/src/apps/linkis/module/engineConfigurationTemplate/index.vue:84]: Suggest increasing internationalization(建议增加国际化)

File: linkis-web/src/apps/linkis/module/engineConfigurationTemplate/index.vue [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+import {getEngineList, queryEngineByID, deleteTemplateByID, changeTemplate} from "./service";
+export default {
+  mixins: [mixin],
+  components: {EditForm},
+  created(){
+    getEngineList().then((data) => {
+      this.engineList = data['success: '];
+      //console.log(this.engineList)
+    })
+    this.queryEngineTemplateList(this.labelId); //默认获取全局设置
```
Comment (jacktao007) [inline linkis-web/src/apps/linkis/module/engineConfigurationTemplate/index.vue:106]: English comments are recommended(建议使用英文注释)

**Review by jacktao007 — DISMISSED** [review]
LGTM.

**Review by jacktao007 — APPROVED** [review]
LGTM.

**Review by peacewong — APPROVED** [review]
LGTM.

**Pull request merged**